packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -60,7 +60,8 @@ export class SearchBoxBase extends BaseComponent<ISearchBoxProps, ISearchBoxStat
       labelText,
       theme,
       clearButtonProps,
-      disableAnimation
+      disableAnimation,
+      clearButtonAriaLabel = 'Clear text'
```

Comment:
I would avoid putting the default in the destructure and put it in React's static `defaultProps` object instead (for consistency and more predictability).